crates/models/src/materializations.rs

```diff
@@ -41,6 +41,13 @@ pub struct MaterializationDef {
     pub delete: bool,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, JsonSchema, PartialEq)]
+#[serde(untagged)]
+pub enum DekafConfigContainer {
```

Comment:
@jgraettinger This seem right to you? The rest of the config objects just use a `RawValue` for their config because it's dynamic, and that lets them inherently accept either a string (which is treated as an indirect reference), or an object (which is treated as an inline config).